drv/lpc55-sprot-server/src/main.rs

```diff
@@ -250,7 +260,7 @@ impl IO {
             if self.txcount >= tx_end || !self.spi.can_tx() {
                 break;
             }
-            let b = self.tx_buf[self.txcount];
+            let b = self.tx_buf.as_slice()[self.txcount];
```

Comment:
Would it be worth it to implement
```
impl<T, I: SliceIndex> Index/IndexMut for TxBuf
```
?

Reply:
Maybe? I'm actually thinking of changing how these types work a bit... Will keep it in mind.